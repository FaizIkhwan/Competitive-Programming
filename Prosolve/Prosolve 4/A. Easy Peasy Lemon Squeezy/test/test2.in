4
-10 10 1
-1 20 2
99 150 2
999 1000 3