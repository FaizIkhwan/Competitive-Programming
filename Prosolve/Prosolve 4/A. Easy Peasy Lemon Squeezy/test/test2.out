-10 -9 -8 -7 -6 -5 -4 -3 -2 -1 0 1 2 3 4 5 6 7 8 9 10 
-1 1 3 5 7 9 11 13 15 17 19 
99 101 103 105 107 109 111 113 115 117 119 121 123 125 127 129 131 133 135 137 139 141 143 145 147 149 
999 
